form.js

```diff
@@ -0,0 +1,58 @@
+
+/**
+ * Creates a menu entry in the Google Docs UI when the document is opened.
+ * This method is only used by the regular add-on, and is never called by
+ * the mobile add-on version.
+ *
+ * @param {object} e The event parameter for a simple onOpen trigger. To
+ *     determine which authorization mode (ScriptApp.AuthMode) the trigger is
+ *     running in, inspect e.authMode.
+ */
+function onOpen() {
+  DocumentApp.getUi() 
+      .createMenu('Form')
```

Comment:
This should be 'File Mapping' ?

Reply:
This is mainly for testing that's why didn't change it.

Reply:
We could still keep useful names for testing as well :)

Reply:
Yeah true !!